packages/models/src/network/network.model.ts

```diff
@@ -177,7 +196,7 @@ const networkSbtcDevenv: NetworkConfiguration = {
       blockchain: 'bitcoin',
       bitcoinNetwork: 'regtest',
       mode: 'regtest',
-      bitcoinUrl: 'http://localhost:8083/api',
+      bitcoinUrl: 'http://localhost:3000/api/proxy',
```

Comment:
Thought: Would we want a potential option to configure this?

Reply:
Yes, totally. 

And users can add networks in extension, _but_ we're missing feature to target those via the API, so for now just an accurate set up will unblock sbtc teams